pkg/resource/v1/certconfig/current_test.go

```diff
@@ -0,0 +1,158 @@
+package certconfig
+
+import (
+	"context"
+	"reflect"
+	"testing"
+
+	"github.com/giantswarm/apiextensions/pkg/apis/core/v1alpha1"
+	"github.com/giantswarm/apiextensions/pkg/clientset/versioned/fake"
+	"github.com/giantswarm/certs"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"k8s.io/apimachinery/pkg/runtime"
+	clientgofake "k8s.io/client-go/kubernetes/fake"
+	k8stesting "k8s.io/client-go/testing"
+)
+
+func Test_GetCurrentState(t *testing.T) {
```

Comment:
Test case for verifying paging implementation is currently missing because `fake.SimpleClientset` doesn't implement paging. I'm currently a bit unsure if it makes sense to simulate this with `Reactor` implementation when it wouldn't necessarily stay in sync with real k8s api implementation.